### ISSUE TITLE
<Question> How about changing file directory

### ISSUE THREAD

**Issue by kaikai529** [Hattomo/kaikai_AI#168]
## Description
we improve independence of codes and refactor file directory

**for example**

kaikai-AI
 - dnn
     - main
     - neural network
 - cnn
     - main + convolution neural network
     - convolution layer
     - pooling layer
     - fully connect layer
- setting
   - parameter(parameter setting file)
   - weight and kernel
   - activation unction
   - loss function
- datasets
   - dnn
   - cnn
- analysis tool
 - tests
## Check list 
- [ ]

**Comment by kaikai529** [comment]
Please give me your opinions.

**Comment by kaikai529** [comment]
done
